compiler/rustc_target/src/spec/loongarch64_unknown_linux_gnu.rs

```diff
@@ -0,0 +1,17 @@
+use crate::spec::{Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "loongarch64-unknown-linux-gnu".into(),
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n64-S128".into(),
+        arch: "loongarch64".into(),
+        options: TargetOptions {
+            cpu: "generic-la64".into(),
+            features: "+f, +d".into(),
```

Comment:
The space after the comma is wrong here (this string is not meant for end user consumption).

```suggestion
            features: "+f,+d".into(),
```

Otherwise it will cause failure later:

```
thread 'rustc' panicked at 'compiler/rustc_codegen_llvm/src/llvm_util.rs:497:9: target feature ` +d` must begin with a `+` or `-`', compiler/rustc_middle/src/util/bug.rs:36:26
```

Reply:
thanks，I also found this problem